src/views/activity/Activity.vue

```diff
@@ -116,7 +116,11 @@ export default {
     host: String,
     date: {
       type: String,
-      default: get_today(),
+      // NOTE: This does not work as you'd might expect since the default is set on
+      // initialization, which would lead to the same date always being returned,
+      // even if the day has changed.
+      // Instead, use the computed _date.
+      //default: get_today(),
```

Comment:
No point in keeping the old code, could just as well be removed.

Reply:
I'd like to keep it as documentation to explain why we aren't using the normal default and why you shouldn't use the date prop directly.